Dashboard/Connectors/Serial/SerialConnector.cs

```diff
@@ -161,4 +198,19 @@ private bool ParseBool(string value)
         return value.StartsWith('#') ? _random.Next() % 2 == 0 : value == "1";
     }
 
+    private void SendHeartbeat(object? state, ElapsedEventArgs e)
+    {
+        if (comPort.IsConnected)
+        {
+
+            Console.WriteLine($"Sending heartbeat: {HeartBeatMessage}");
+            comPort.Write(HeartBeatMessage);
```

Comment:
For now we could have some fake implementation that returns the `OK` message when we do a connection test?

Reply:
so we can implement the handling of that now rather than later